classes/opponentBase.lua

```diff
@@ -20,6 +20,8 @@ function opponentBase.new(implClass)
 	self.stunned = false
 	self.stunTime = 0
 
+	self.intangible = false 	-- Set to true if it shouldn't collide with player
```

Comment:
Created new property called "intangible" for the opponentBase - if it's set to true, then the player WON'T take damage when touching it